src/NexusMods.App.UI/ImagePipelines.cs

```diff
@@ -256,11 +258,15 @@ private static IResourceLoader<Uri, Lifetime<Bitmap>> CreateGuidedInstallerRemot
         return pipeline;
     }
 
-    private static IResourceLoader<Uri, Bitmap> CreateMarkdownRendererRemoteImagePipeline(HttpClient httpClient)
+    private static IResourceLoader<Uri, IImage> CreateMarkdownRendererRemoteImagePipeline(HttpClient httpClient)
     {
         var pipeline = new HttpLoader(httpClient)
-            .Decode(decoderType: DecoderType.Skia)
-            .ToAvaloniaBitmap();
+            .EnableSvgSupport(static inner => inner
+                .Decode(decoderType: DecoderType.Skia)
+                .ToAvaloniaBitmap()
+                .UseFallbackValue(MarkdownFallback)
+            )
+            .UseFallbackValue(MarkdownFallback);
```

Comment:
This looks a bit strange, it isn't clear whether the `Decode` and `ToAvaloniaBitmap` steps always happen or just for Svgs